docs/content/en/docs/install/install.md

```diff
@@ -71,228 +38,172 @@ The command sequence to fetch and install the latest release of Keptn is:
-
-### Enable Keptn Analysis (SLOs/SLIs)
-
-To enable Keptn Analysis in your cluster, you again do not need the Keptn Lifcycle Operator,
-disable it using the following values.yaml:
-
-{{< embed path="/docs/content/en/docs/install/assets/values-only-metrics.yaml" >}}
-
-> **Note** The Analysis use-case is currently behind a feature flag.
-  To enable it, add the following to your `helm upgrade` command line:
-
-  ```shell
-  --set metricsOperator.env.enableKeptnAnalysis=true
-  ```
->
-
-For more information about implementing Keptn Analysis, see the
-[Analysis User Guide](../implementing/slo.md).
-
 ### Disable Keptn Certificate Manager (Certificates)
 
 If you wish to use your custom certificate manager,
-you can disable Keptn `cert-manager` by using the
-`--set " certificateManager.enabled=false"` argument
-to the `helm upgrade` command line
-or you can modify the `keptn/values.yaml` file:
+you can disable Keptn `cert-manager` by setting the
+`certificateManager.enabled` Helm value to `false`:
 
-{{< embed path="/docs/content/en/docs/install/assets/values-remove-certmanager.yaml" >}}
+{{< docsembed path="content/en/docs/install/assets/values-remove-certmanager.yaml" >}}
 
 For more information on using `cert-manager` with Keptn, see
 [Use Keptn with cert-manager.io](../operate/cert-manager.md).
 
-For more advanced installations configurations,see:
-
-* [CertificateManager-README](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-cert-manager/README.md)
-
-## Customizing the configuration of components
-
-To access and modify the configuration of a subcomponent,
-modify the component's `values.yaml` file.
-You can use the sub-chart name
-as written in the `chart.yaml` file.
-The `README` file for each component documents the parameters
-that are supported for that component.
-
-Here is an example `values.yaml` altering metrics operator values:
-
-{{< embed path="/docs/content/en/docs/install/assets/values-advance-changes.yaml" >}}
-
-## Running Keptn with vCluster
-
-Keptn running on Kubernetes versions 1.26 and older
-uses a custom
-[scheduler](../architecture/components/scheduler.md),
-so it does not work with
-[Virtual Kubernetes Clusters](https://www.vcluster.com/)
-("vClusters") out of the box.
-This is also an issue
-if the `schedulingGatesEnabled` Helm chart value is set to `false`
-for Kubernetes version 1.27 and later.
-See
-[Keptn integration with Scheduling](../architecture/components/scheduler.md)
-for details.
-
-To solve this problem:
-
-1. Follow the instructions in
-   [Separate vCluster Scheduler](https://www.vcluster.com/docs/architecture/scheduling#separate-vcluster-scheduler)
-   to modify the vCluster `values.yaml` file
-   to use a virtual scheduler.
-
-1. Create or upgrade the vCluster,
-   following the instructions in that same document.
-
-1. Follow the instructions in the section below
-   to install Keptn in that vCluster.
+For the full list of Helm values, see the
```

Comment:
Should we delete this link.  It is in the table above and the configuration described here is not in the cert-manager Helm chart.  Moreover, this tells them how to disable the Keptn cert-manager so a bit confusing to now tell them how to set Helm values for it.  If you want to leave this link, could modify the read-in, for example: "To control the specific behavior of the default Keptn cert-manager, modify the Helm values described in:"